src/Tendermint/Serialization.h

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../proto/Cosmos.pb.h"
+#include <nlohmann/json.hpp>
+
+namespace TW {
+namespace Cosmos {
```

Comment:
`namespace TW::Cosmos`

Reply:
> `namespace TW::Cosmos`

done in ce68b6caed56b6982d81ffe3f571e9477b757ad8